### DISCUSSION TITLE
[Nhận TASK]

### DISCUSSION THREAD

**Discussion by trungdoublelift** [Runway-Club/Runway-CV-Builder#49] [Task Delivery]
Tôi muốn nhận task : Tạo thuộc tính cho 1 user : tài khoản , mật khẩu , tên (full name) , CV List ( các cv mẫu đã tạo) , List tên các doanh nghiệp đã nộp đơn.

**Comment by nguyentantrung2000** [comment]
Sau khi xem xét thì task này vẫn chưa có người làm. Vậy bạn sẽ nhận task này nhé ! 
Thông tin task:
-Thuộc tính của user để dễ dàng quản lý.
Yêu cầu:
-Phải đầy đủ các trường đã yêu cầu
-Deadline 11:59PM 5/8/2021

**Reply by trungdoublelift** [reply]
OK
